src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -15,7 +15,11 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     public static partial class SymbolFinder
     {
         /// <summary>
-        /// Finds the symbol that is associated with a position in the text of a document.
+        /// Finds the symbol that is associated with a position in the text of a document. If 
+        /// multiple symbols are associated with the token (e.g. implicitly named anonymous type
```

Comment:
What about the case in VB where queries both reference a variable and define a variable? Did we change that behavior?